### ISSUE TITLE
Broken link about IAM users in “Getting Started with Convox” page

### ISSUE THREAD

**Issue by jjperezaguinaga** [convox/convox.github.io#46]
*tl;dr: Broken link in production, good link in source code, deploy to fix.*

The following link points to this [broken URL](http://docs.convox.com/docs/creating-an-iam-user-and-credentials) instead of the valid [URL](http://docs.convox.com/docs/creating-an-iam-user/)

![screen shot 2015-09-23 at 09 39 07](https://cloud.githubusercontent.com/assets/1128312/10039909/f0d3a9f4-61d6-11e5-948c-b962778efd97.png)

I tried to send a PR but seems the [file](https://github.com/convox/convox.github.io/blob/master/_getting_started/getting-started-with-convox.md) has the actual URL, so you guys just need to do a quick deploy of the docs to be fixed :)
